### PR TITLE
Fix PR merge script.

### PR DESCRIPTION
**Pull request by Zimmi48**
@maximedenes Thanks for having shared your merge script. I used it to merge a PR today for the first time. But first I needed to fix a few things.
I believe you have some specific configuration in your `.git/config` or in your `.gitconfig` which defines what `github/pr/XXX` is. Given that I didn't have this, the script was failing but I think that my proposed fix should still work for you.

### CODE REVIEW

**Comment by maximedenes** [comment]
Btw, merging this PR is a bit tricky because it updates the script as it is running...

File: dev/tools/merge-pr.sh [hunk 1]
```diff
@@ -15,12 +15,11 @@ API=https://api.github.com/repos/coq/coq
 
 BASE_BRANCH=`curl -s $API/pulls/$PR | jq -r '.base.label'`
 
-COMMIT=`git rev-parse $REMOTE/pr/$PR`
```
Comment (maximedenes) [inline dev/tools/merge-pr.sh]: This breaks the line coming just after...

**Review by maximedenes — DISMISSED** [review]

File: dev/tools/merge-pr.sh (hunk 1 above)
Comment (Zimmi48) [reply dev/tools/merge-pr.sh]: Fixed.

**Comment by Zimmi48** [comment]
I also changed the shebangs to make the use of `bash` explicit given that this script apparently relies on bash-specific features (see https://github.com/coq/coq/pull/6271#discussion_r153632223).

**Comment by maximedenes** [comment]
Strange, this PR was not automatically closed, it seems.

**Comment by Zimmi48** [comment]
To me it appears as merged.

**Comment by maximedenes** [comment]
Merged yes, but not closed. I closed it by hand.

**Pull request merged**